toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -511,10 +511,6 @@ Example: `noatime,nodiratime`
 Specifies the configuration for dm-verity root integrity verification. Please
 execute `sudo modprobe nbd` before building the image with verity enablement.
```

Comment:
While you're in the area, it might be goodness to remove the reference to `nbd`.